state/execution.go

```diff
@@ -292,6 +292,11 @@ func execBlockOnProxyApp(
 
 	commitInfo := getBeginBlockValidatorInfo(block, store, initialHeight)
 
+	var byzVals []abci.Evidence //nolint:prealloc
```

Comment:
why not preallocate?

Reply:
because we have no idea what the size of the array will be

Reply:
Yeah, seems like the linter doesn't handle the case of using `...` inside a loop. But we may at least wait to initialize it to an empty slice, to avoid any `nil` slice issues.